schemas/preferences.sublime-settings.json

```diff
@@ -1211,6 +1211,42 @@
           "type": "array",
           "items": {"type": "string"},
           "default": ["Vintage"]
+        },
+        "default_extension": {
+          "markdownDescription": "Set's the default extension in the OS Save dialog when an untitled file is saved.",
```

Comment:
I added prefixes like `**Experimental**` to "mini_auto_complete". Would it make sense to add something like `**Unlisted**` to these unlisted settings?

Reply:
I have added **Unlisted** to all of these settings. I guess **Experimental** might give the wrong idea that the setting is unstable for use and might go away in a future update (which I don't believe is the case with `mini_auto_complete`)

Also for references purposes 
1. `print_using_browser` is defined in `Default/html_print.py`
2. `show_legacy_color_schemes` is defined in `Default/ui.py`
3. `default_extension` is defined in `Default/new_templates.py`

Looks like I also forgot to add `default_dir` (defined in `Default/new_templates.py`). https://github.com/sublimelsp/LSP-json/pull/91/commits/9330c2804b345e80113a22046c23a923be09dc73 should take care of it.